README.md

```diff
@@ -30,25 +38,37 @@ usage: agentmon [flags] sink-URL
         print version string
 ```
 
-#### Run Tests
+## Developing
+
+### Run Tests
+
 ```bash
 make test 
 ``` 
-#### Install
+
+### Install
+
 ```bash
 make install 
 ```
-#### Build
+
+### Build
+
 ```bash
 make build 
 ```
-#### Create Release Package 
+
+### Create Release Package 
+
 ```bash
 make release 
 ```
 
-
-
 ## Copyright
 
 (c) 2017, Heroku, Inc. See [LICENSE](./LICENSE) for details.
```

Comment:
Do we need to update this for '18?

Reply:
Uh. I'm not actually sure. Presumably, yes.

Reply:
No, you don't. The date should be the date of first publication and it establishes the start of the copyright.